.changeset/friendly-mayflies-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-site": major
+---
+
+Revise `PixelImageBlock` to use updated next image component (remove layout prop)
```

Comment:
Changesets for breaking changes should be detailed. What do I need to do to upgrade? Maybe link to the Next upgrade guide (if it exists)

Reply:
i updated the changeset.